src/pcapi/routes/pro/venues.py

```diff
@@ -89,6 +90,30 @@ def edit_venue(venue_id: str, body: EditVenueBodyModel) -> GetVenueResponseModel
     venue_attrs = body.dict(exclude=not_venue_fields, exclude_unset=True)
     venue = offerers_api.update_venue(venue, body.contact, **venue_attrs)
 
+    accessibility_changes = {
+        field: body[field] != venue[field]
+        for field in [
+            "audioDisabilityCompliant",
+            "mentalDisabilityCompliant",
+            "motorDisabilityCompliant",
+            "visualDisabilityCompliant",
+        ]
+    }
+    if True in accessibility_changes and body.isAccessibilityAppliedOnAllOffers:
+        edit_accessibility = {
+            field: body[field] != venue[field]
```

Comment:
```suggestion
            field: body[field]
```

Sinon on updatera la valeur du champ à `True` même si on la passe de `True` à ` False`